src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalSum = days * 40;
+
+  if (days < 3) {
+    return totalSum;
+  }
+
+  if (days >= 3 && days < 7) {
+    return totalSum - 20;
+  }
+
+  if (days >= 7) {
+    return totalSum - 50;
+  }
```

Comment:
redundant `if` statement
```suggestion
  return totalSum - 50;
```